hs_file_types/tests/test_generic_metadata.py

```diff
@@ -342,8 +346,9 @@ def test_aggregation_xml_file_paths(self):
                                       src_path, tgt_path)
         res_file = self.composite_resource.files.first()
         logical_file = res_file.logical_file
-        expected_meta_path = '{0}/{1}_meta.xml'.format(folder_rename, res_file.file_name)
-        expected_map_path = '{0}/{1}_resmap.xml'.format(folder_rename, res_file.file_name)
+        res_file_name, _ = os.path.splitext(res_file.file_name)
+        expected_meta_path = '{0}/{1}_meta.xml'.format(folder_rename, res_file_name)
+        expected_map_path = '{0}/{1}_resmap.xml'.format(folder_rename, res_file_name)
```

Comment:
There are so many instances of hardcoded _meta.xml and _resmap.xml in so many test files and also in create xml doc methods above. I am thinking it is probably worth the effort to put this expected metadata and resmap file name string in a centralized place defined in a property or variable that can be referred to in multiple places, so if in the future it were decided to change the file name pattern, we only need to update one or two places rather than so many places. Just a thought for your reference.

Reply:
Fixed.